### PR TITLE
Upgrade Rails version 4 to 5 

### PR DESCRIPTION
**Pull request by ggomagundan**
1. Upgrade rails version
- Rails4 => Rails 5.0

~~2. Upgrade Bootstrap Version~~
- ~~Bootstrap v3 => Bootstrap v4.alpha~~

 3. Change Root Path
- plainpage#index  => plainpage#dashborad

4. Update Gentelella Version
- update Gentelella version(1.1.0 to 1.3.0) and update vendor files

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -43,7 +52,7 @@ end
 
 group :development do
   # Access an IRB console on exception pages or by using <%= console %> in views
-  gem 'web-console', '~> 2.0'
+  gem 'web-console', '~> 3.0'
```
Comment (PeterTeng) [inline Gemfile]: This is the sample project for Gentelella only.
It's better to just add necessary Gems

File: Gemfile.copy [hunk 2]
```diff
@@ -0,0 +1,110 @@
+source 'https://rubygems.org'
```
Comment (PeterTeng) [inline Gemfile.copy]: It's better to clean up comment outs🤔

File: Gemfile [hunk 3]
```diff
@@ -18,6 +20,13 @@ gem "font-awesome-rails"
 
 gem 'bootstrap-sass', '~> 3.3.6'
 
+gem 'bootstrap', '~> 4.0.0.alpha5'
```
Comment (PeterTeng) [inline Gemfile]: I think gentelella is not using bootstrap above version 4.x🤔
It may be better to match the version

**Review by PeterTeng — CHANGES_REQUESTED** [review]
I just made some reviews😬

I think it is better to separate the pull request in each features

By each feature should provide the reason of changes.

File: Gemfile.copy (hunk 2 above)
Comment (ggomagundan) [reply Gemfile.copy]: @PeterTeng  Oh, This is my mistake :( , I Committing remove this file

File: Gemfile (hunk 3 above)
Comment (ggomagundan) [reply Gemfile]: @PeterTeng  Now, I downgrade Bootstrap Version 3 :) Thx to your Reviews

**Comment by ggomagundan** [comment]
After Review

- Remove `Gemfile.copy` File (For Reference Rails5 Project Gemfile )
- Change stylesheet mismatch path
- Remove duplicate `require` line ( duplicate add `js/custom.js` file)

**Pull request closed without merging**